src/main/java/no/nav/familie/inntektsmelding/forespørsel/rest/ForespørselRest.java

```diff
@@ -180,6 +182,20 @@ public Response sendNyBeskjedOgVarsel(@Valid @NotNull NyBeskjedRequest request)
         return Response.ok().build();
     }
 
+    @GET
+    @Path("/sak")
+    @Tilgangskontrollert
+    public Response hentForespørslerForSak(@Valid @NotNull SaksnummerDto saksnummer) {
```

Comment:
Mulig vi også burde sende med aktørId for å gjøre en ekstra sikkerhetssjekk. Nå burde jo saksnummer være unikt for aktørID, men muligens noe å tenke på likevel